assets/js/blocks/product-categories/block.js

```diff
@@ -142,6 +152,24 @@ const ProductCategoriesBlock = ( { attributes, setAttributes, name } ) => {
 							setAttributes( { hasEmpty: ! hasEmpty } )
 						}
 					/>
+					{ isSiteEditor && (
```

Comment:
This option will be available when adding the block to any template in the site editor, even if it's not the Product Categories one (ie: when adding the block to the _Home_ template). I don't know if this is an issue, but I wanted to mention just in case.

Another thing to consider is whether we should enable this option when adding the block to the _Widgets_ screen in classic themes.

Reply:
>I don't know if this is an issue, but I wanted to mention just in case.

Yeah, this is a known issue which is why I put the tooltip in the `help` prop on the editor side. I didn't want to overly complicate this since I don't think there's a standard way to approach this. We'd also need to consider all kinds of category templates e.g. category specific ones.

>Another thing to consider is whether we should enable this option when adding the block to the Widgets screen in classic themes.

Yes, good point! I will address this in this PR.

Reply:
Done in https://github.com/woocommerce/woocommerce-blocks/pull/9550/commits/73008c0a138ea3cf1f7f3d1910f2d7833b1e1488